### ISSUE TITLE
vllm0.9.2 v1不支持logits processors

### ISSUE THREAD

**Issue by 741292662** [PKU-YuanGroup/Edit-R1#9]
请问有人遇到0.9.2的版本 进行评估的时候显示ValueError: vLLM V1 does not support per request user provided logits processors.的吗
